src/main/java/org/jitsi/videobridge/health/Health.java

```diff
@@ -182,7 +182,7 @@ public static void check(Videobridge videobridge)
 
         if (cachedExceptionTimestamp > 0 &&
```

Comment:
will the code be simpler if you include `cachedException != null` as the first `if` condition ?

Reply:
No, actually not. It would not be correct

Reply:
No, I want to finish execution (either throw or return) regardless of the value of cachedException

Reply:
right